### PR TITLE
Issue 1133, update Swift 3 container and tests

### PR DESCRIPTION
**Pull request by paulcastro**
This updates the Swift 3 runtime and associated tests.  It does not update the default Swift container to be Swift 3.

### CODE REVIEW

**Comment by paulcastro** [comment]
PG #544 successful.

File: tests/src/actionContainers/SwiftActionContainerTests.scala [hunk 1]
```diff
@@ -33,6 +33,21 @@ class SwiftActionContainerTests extends BasicActionRunnerTests with WskActorSyst
     val enforceEmptyOutputStream = true
     lazy val swiftContainerImageName = "swiftaction"
 
+    lazy val envCode = """
```
Comment (rabbah) [inline tests/src/actionContainers/SwiftActionContainerTests.scala]: is this any different than swift3 version?

**Comment by rabbah** [comment]
Thanks. I missed that one.

**Comment by bjustin-ibm** [comment]
LGTM

File: tests/src/actionContainers/Swift3ActionContainerTests.scala [hunk 2]
```diff
@@ -98,16 +103,16 @@ class Swift3ActionContainerTests extends SwiftActionContainerTests {
 
         checkStreams(out, err, {
             case (o, e) =>
-                o shouldBe empty
+                //o shouldBe empty
```
Comment (psuter) [inline tests/src/actionContainers/Swift3ActionContainerTests.scala]: Why is this now commented out?

File: tests/src/actionContainers/Swift3ActionContainerTests.scala [hunk 3]
```diff
@@ -125,7 +130,7 @@ class Swift3ActionContainerTests extends SwiftActionContainerTests {
...
```
Comment (psuter) [inline tests/src/actionContainers/Swift3ActionContainerTests.scala]: Same question.

File: core/swift3Action/spm-build/Whisk.swift [hunk 4]
```diff
@@ -124,31 +129,31 @@ class Whisk {
...
-
+            
             request.write(from: jsonData)
             request.end()
         } catch {
             callback(["error": "Could not parse parameters."])
         }
     }
-
+
```
Comment (psuter) [inline core/swift3Action/spm-build/Whisk.swift]: Trailing white space everywhere 🙀 (not enforced one `*.swift` files by `scanCode.py`, so no reason to fix now).

**Comment by psuter** [comment]
Also: you may want to write something like `Fixes #1133` explicitly in the commit message to make GitHub dark magic work: https://help.github.com/articles/closing-issues-via-commit-messages/

File: core/swift3Action/spm-build/Whisk.swift (hunk 4 above)
Comment (paulcastro) [inline core/swift3Action/spm-build/Whisk.swift]: The package manager prints lots of things to std out and there doesn't seem to be a way to quiet it.

File: tests/src/actionContainers/Swift3ActionContainerTests.scala (hunk 2 above)
Comment (paulcastro) [inline tests/src/actionContainers/Swift3ActionContainerTests.scala]: The swift package manager likes to print out what its doing and there doesn't seem to be a way to quiet it.

**Pull request merged**